### PR TITLE
Localized welcome email for music research lab.

### PR DESCRIPTION
**Pull request by yrammos**

### CODE REVIEW

File: translations/config.json [hunk 1]
```diff
@@ -90,7 +90,8 @@
         "e-alfred ocdownloader",
         "e-alfred flowupload",
         "e-alfred nextcloud-scanner",
-        "affan98 forms"
+        "affan98 forms",
```
Comment (skjnldsv) [inline translations/config.json]: @jancborchardt @rullzer  I suddenly realise we should rename this to nextcloud/forms, right?

**Comment by skjnldsv** [comment]
Hey  @yrammos 
I'm not sure this is the right place.
Is your app released to the store? IS this going to be for your own use only?

**Comment by yrammos** [comment]
@skjnldsv This is not intended for app store release. If this is not the appropriate community, I stand corrected and will withdraw the PR. Kindly let me now. Thank you.

**Comment by skjnldsv** [comment]
> @skjnldsv This is not intended for app store release. If this is not the appropriate community, I stand corrected and will withdraw the PR. Kindly let me now. Thank you.

I'll suggest you add your translations in your own repository then :)

**Comment by yrammos** [comment]
Apologies for misunderstanding, @skjnldsv. Will do.

**Comment by skjnldsv** [comment]
> Apologies for misunderstanding, @skjnldsv. Will do.

No worries!
Glad to see people developing their nextcloud apps :)

File: translations/config.json (hunk 1 above)
Comment (jancborchardt) [reply translations/config.json]: Yup! The repository is moved and https://github.com/affan98/forms redirects to https://github.com/nextcloud/forms :)

**Pull request closed without merging**